packages/ui/src/components/Logo/Logo.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack } from '../Stack';
+import { Logo, LogoProps } from './Logo';
+
+const meta: Meta = {
+  title: 'Components / Logo',
+  component: Logo,
+};
+
+export default meta;
+
+const Template: Story<LogoProps> = (args) => <Logo {...args} />;
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  layout: 'horizontal',
+  color: 'black',
+};
+
+export const LayoutVariants = () => (
```

Comment:
You can name it just `Layouts` as you did in the case of colors stories.

Reply:
done